learning-ojs/en/site-administration.md

```diff
@@ -93,6 +99,15 @@ Use this section to modify your site-wide menus. You can learn more about config
 
 ![](./assets/learning-ojs3.1-sa-menus.png)
 
+####  Bulk Emails
+OJS 3.3 has the option to allow bulk emails to be sent to several user groups in a journal at one time. See [Users and Roles > Email Users](/users-and-roles#email-users). As the site administrator, you can use the site settings to select the hosted journals that will be allowed to send bulk emails.
+
+Note that misuse of this figure to send unsolicited email may violate anti-spam laws in some jurisdictions and may result in your server’s emails being blocked as spam. Seek technical advice before enabling this feature and consider consulting with journal managers to ensure it is used appropriately.
+
+![](./assets/learning-ojs3.3-site-settings-bulk-emails.png)
+
+It is possible to disable sending bulk emails to certain roles for individual journals under [Hosted Journals](site-administration#hosted-journals) > Settings Wizard.
```

Comment:
Since this is linking to a section within the same page, it should be formatted as [Hosted Journals](#hosted-journals).